manager/integration/tests/common.py

```diff
@@ -3287,15 +3287,20 @@ def expand_attached_volume(client, volume_name):
 
 def prepare_pod_with_data_in_mb(
         client, core_api, pod_make, volume_name, volume_size=str(1*Gi),
-        data_path="/data/test", data_size_in_mb=DATA_SIZE_IN_MB_1):  # NOQA:
+        data_path="/data/test", data_size_in_mb=DATA_SIZE_IN_MB_1,
+        add_liveness_prope=True):  # NOQA:
```

Comment:
The title of the commit is wrong. This part is not a fix for a bug.

Reply:
fixed